staking/stake.md

```diff
@@ -87,9 +102,14 @@ To nominate a collator, provide the following information:
  3. Choose the extrinsic method to use for the transaction. This will determine the fields that need to fill in the following steps. In this case, it is the `nominate` extrinsic
  4. Set the collator's address you want to nominate. In this case, it is set to `{{ networks.moonbase.staking.collators.address1 }}`
  5. Set the number of tokens you want to stake
- 6. Click the "Submit Transaction" button and sign the transaction
+ 6. Input the `collator_nominator_count` you retrieved above from the JavaScript console
+ 7. Input the number of previous nominations you have made from your selected account. This would be 0 if you haven't yet nominated a collator. If you are uncertain of how many nominations you have already made, estimate a maximum number
```

Comment:
Is it possible to get a value using javascript?

Reply:
Yes! Updated with the JS query.